x/staking/client/rest/utils.go

```diff
@@ -50,7 +50,7 @@ func queryBonds(cliCtx context.CLIContext, cdc *codec.Codec, endpoint string) ht
 			return
 		}
 
-		params := staking.NewQueryBondsParams(delegatorAddr, validatorAddr)
+		params := querier.NewQueryBondsParams(delegatorAddr, validatorAddr)
```

Comment:
shouldn't these params live in `x/staking/types/querier.go` ?

Reply:
yeah - I didn't want to refactor the querier within staking to live in types and keeper... but I just did b/c of this comment... thanks!